### PR TITLE
Changed the lscorecard.py

### PR DESCRIPTION
**Pull request by yashrsharma44**
Changed the function getScoreCard.

### CODE REVIEW

File: CrickFev/scripts/lscorecard.py [hunk 1]
```diff
@@ -9,124 +9,104 @@
...
+        batsman_data.append(tds)
+
+    for rows in details[1]:
+        tds = rows.find_all("td")
+        bowler_data.append(tds)
+
+
+    
+    if len((batsman_data[0])[0])<25:
+    		(batsman_data[0])[0] +=" "*(25-len((batsman_data[0])[0]))
```
Comment (sipah00) [inline CrickFev/scripts/lscorecard.py]: change tab to space

File: CrickFev/scripts/lscorecard.py [hunk 2]
```diff
@@ -9,124 +9,104 @@
...
+        if len(sr)<4:
+            sr +=" "*(4-len(sr))
+
+        print(playerName + "\t"+playerScore+"\t"+ballsFaced+"\t"+fours+"\t"+six+"\t"+sr)
+
+    print("\n"+"*"*70)
+
+    
+    if len((bowler_data[0])[0])<25:
+    		(bowler_data[0])[0] +=" "*(25-len((bowler_data[0])[0]))
```
Comment (sipah00) [inline CrickFev/scripts/lscorecard.py]: change tab to space

File: CrickFev/scripts/lscorecard.py [hunk 3]
```diff
@@ -9,124 +9,104 @@
...
+        maid +=" "*(4-len(maid))
+        runs +=" "*(4-len(runs))
+        wkt  +=" "*(4-len(wkt))
+        eco +=" "*(4-len(eco))
+
+        print(name +"\t"+over+"\t"+maid+"\t"+runs+"\t"+wkt+"\t"+eco)
+        
+
+        
+
```
Comment (sipah00) [inline CrickFev/scripts/lscorecard.py]: remove some blank lines

**Comment by sipah00** [comment]
@yashrsharma44, Nice work, I have suggested some changes, go through it.

**Comment by yashrsharma44** [comment]
Thanks for your suggestions. I have removed excess spaces.

**Pull request merged**